### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by PowershellScripts**
Added missing parameters: InformationBarriersSuspension, IBImplicitGroupBased, DefaultOneDriveInformationBarrierMode.

The parameters are already available in the released SharePoint Online Management Shell v 16.0.22810.
The descriptions are based on https://learn.microsoft.com/en-us/purview/information-barriers-sharepoint.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -534,7 +537,7 @@ The valid values are:
 - False - No folder is created when the site and OneDrive for Business document library is created.
 
 The default behavior of the Shared with Everyone folder changed in August 2015.  
-For additional information about the change, see Provision the Shared with Everyone folder in OneDrive for Business (<https://support.office.com/en-us/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2?ui=en-US&rs=en-US&ad=US)>
+For additional information about the change, see [Provision the Shared with Everyone folder in OneDrive for Business](https://support.office.com/en-us/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2?ui=en-US&rs=en-US&ad=US)
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:540]:
```suggestion
For additional information about the change, see [Provision the Shared with Everyone folder in OneDrive for Business](https://support.office.com/en-us/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2?ui=en-US&rs=en-US&ad=US).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2314,6 +2317,64 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IBImplicitGroupBased
+
+The IBImplicitGroupBased setting enables Microsoft 365 group-membership based access and sharing control for all Implicit mode sites.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2338]:
```suggestion
The IBImplicitGroupBased setting enables Microsoft 365 Groups membership-based access and sharing control for all Implicit mode sites.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -534,7 +537,7 @@ The valid values are:
 - False - No folder is created when the site and OneDrive for Business document library is created.
 
 The default behavior of the Shared with Everyone folder changed in August 2015.  
-For additional information about the change, see Provision the Shared with Everyone folder in OneDrive for Business (<https://support.office.com/en-us/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2?ui=en-US&rs=en-US&ad=US)>
+For additional information about the change, see [Provision the Shared with Everyone folder in OneDrive for Business](https://support.office.com/en-us/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2?ui=en-US&rs=en-US&ad=US).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:540]:
```suggestion
For additional information about the change, see [Provision the Shared with Everyone folder in OneDrive for Business](https://support.office.com/article/Provision-the-Shared-with-Everyone-folder-in-OneDrive-for-Business-6bb02c91-fd0b-42ba-9457-3921cb6dc5b2).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -2314,6 +2317,64 @@ Accept pipeline input: False
...
+
+The valid values are:
+
+- Open
+- Explicit
+- Implicit
+- OwnerModerated
+- Mixed
+
+For more information about information barriers, see [Use information barriers with SharePoint](https://learn.microsoft.com/en-us/purview/information-barriers-sharepoint) for your SharePoint online environment.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2364]:
```suggestion
For more information about information barriers, see [Use information barriers with SharePoint](/purview/information-barriers-sharepoint) for your SharePoint online environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @PowershellScripts, two more, please.

**Comment by PowershellScripts** [comment]
@JohanFreelancer9  Happy to. Thank you.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 5]
```diff
@@ -2314,6 +2317,64 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -InformationBarriersSuspension
+
+The InformationBarriersSuspension setting enables information barriers in SharePoint and OneDrive when set to $false, or disables information barriers when set to $true.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2322]: Somehow I think it reads better like this

_When InformationBarriersSuspension parameter is set to $false, information barriers in SharePoint and OneDrive is enabled, when set to $true, it is disabled._ 

@JohanFreelancer9 what do you think?

**Review by JohanFreelancer9 — APPROVED** [review]
I"m good after @get-itips's suggestions.

**Comment by yogkumgit** [comment]
@PowershellScripts

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 5 above)
Comment (PowershellScripts) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: Updated.

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**